deployment/ccip/changeset/state.go

```diff
@@ -6,10 +6,13 @@ import (
 
 	"github.com/smartcontractkit/ccip-owner-contracts/pkg/gethwrappers"
 
-	burn_mint_token_pool "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/burn_mint_token_pool_1_4_0"
```

Comment:
Is this okay? Or do we need this to be bound to 1.4.0 burn mint tokens for some other reason?